### PR TITLE
Git stats input

### PR DESCRIPTION
**Pull request by miguelgfierro**
### Description

Github tracker now accepts dates

### Checklist:

- [ ] My code follows the code style of this project, as detailed in our [contribution guidelines](../CONTRIBUTING).
- [ ] I have added tests.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: scripts/repo_metrics/track_metrics.py [hunk 1]
```diff
@@ -50,9 +52,27 @@ def parse_args():
         action="store_true",
         help="Whether or not to save the information to the database",
     )
+    parser.add_argument(
+        "--event_date",
+        type=_valid_date,
+        default="now",
+        help="Date for an event (format: YYYY-MM-DD or now)",
+    )
```
Comment (gramhagen) [inline scripts/repo_metrics/track_metrics.py]: another option to reduce code would be to leverage dateutil library and remove the _valid_date function
```
from dateutil.parser import isoparse
from datetime import datetime
...
parser.add_argument('--event_date', default=datetime.now().isoformat(), type=isoparse)
```

Comment (miguelgfierro) [reply scripts/repo_metrics/track_metrics.py]: this is so nice! great catch!

**Comment by gramhagen** [comment]
also, do we have pull requests in our current stats? if not is it possible to pull them in?

**Comment by miguelgfierro** [comment]
Good suggestion, it looks that the most efficient way of getting PRs and issue numbers is:
```
https://api.github.com/search/issues?q=state%3Aopen+repo:microsoft/recommenders+type%3Apr
https://api.github.com/search/issues?q=state%3Aopen+repo:microsoft/recommenders+type%3Aissues

```
and get total_count

**Pull request merged**